------------------------------------------------------------------------
r14937 | kostya | 2008-08-24 07:47:45 +0200 (So, 24. Aug 2008) | 4 lines

Correct scaling factor in DCA decoder synthesis.
Patch by Alexander Patrakov ($lastname at gmail.com) 
Thread: dca.c: output scale is a bit off

------------------------------------------------------------------------
Index: libavcodec/dca.c
===================================================================
--- a/libavcodec/dca.c	(Revision 14936)
+++ b/libavcodec/dca.c	(Revision 14937)
@@ -990,7 +990,7 @@
 /*        static float pcm_to_double[8] =
             {32768.0, 32768.0, 524288.0, 524288.0, 0, 8388608.0, 8388608.0};*/
          qmf_32_subbands(s, k, subband_samples[k], &s->samples[256 * k],
-                            2.0 / 3 /*pcm_to_double[s->source_pcm_res] */ ,
+                            M_SQRT1_2 /*pcm_to_double[s->source_pcm_res] */ ,
                             0 /*s->bias */ );
     }
 
